components/automate-ui/src/app/pages/+compliance/+scanner/state/scanner.effects.ts

```diff
@@ -192,7 +192,7 @@ export class ScannerEffects {
     }),
     map(() => new CreateNotification({
       type: Type.info,
-      message: 'Deleted a scan job.'
+      message: 'Deleted scan job.'
```

Comment:
is ID of Deleted scan job needed here ? if yes, we can add a todo for later, here also.

Reply:
Yes, should mark with TODO. Every such message should have an ID.